.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+ updates:
+   - package-ecosystem: "docker"
+     directory: "/"
+     schedule:
+       interval: "daily"
+     open-pull-requests-limit: 10
+     pull-request-branch-name:
+       separator: "-"
+     reviewers:
+       - "teams/rancher-security"
```

Comment:
Do we want to use this team or something like `teams/rancher-team-1-neo-dev`

Reply:
I think that for the moment `teams/rancher-security` is fine, so we don't overwhelm the entire team. If this trial process works fine and we expand it, then adding `teams/rancher-team-1-neo-dev` team is the logical step.